missilewars-plugin/src/main/java/de/butzlabben/missilewars/MessageConfig.java

```diff
@@ -34,30 +34,47 @@
  */
 public class MessageConfig {
 
-    private static final File dir = new File(MissileWars.getInstance().getDataFolder(), "missiles/");
+    private static final File dir = MissileWars.getInstance().getDataFolder();
     private static final File file = new File(MissileWars.getInstance().getDataFolder(), "messages.yml");
     private static YamlConfiguration cfg;
 
     public static void load() {
-        if (!file.exists()) {
+
+        // check if the directory "/MissileWars" is exist
```

Comment:
Check you grammar, it has to be either "check if ... does exist" or "check if ... exists"